test/utils/integration/integration.go

```diff
@@ -0,0 +1,233 @@
+package integration
+
+import (
+	"context"
+	"github.com/avast/retry-go/v3"
+	eventingv1alpha1 "github.com/kyma-project/eventing-manager/api/v1alpha1"
+	"github.com/kyma-project/eventing-manager/internal/controller/eventing"
+	eventingcontroller "github.com/kyma-project/eventing-manager/internal/controller/eventing"
+	"github.com/kyma-project/eventing-manager/test"
+	"github.com/onsi/gomega"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	k8stypes "k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/record"
+	"log"
+	"path/filepath"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	"testing"
+	"time"
+)
+
+const (
+	useExistingCluster       = false
+	attachControlPlaneOutput = false
+	testEnvStartDelay        = time.Minute
```

Comment:
(nit) IMO 1 minutes is long, I think 10-20 seconds should be good.

Reply:
Done